.gitignore

```diff
@@ -1,15 +1,15 @@
 /work-dir/
-/env/certs.env
+/container-host-files/etc/scf/config/env/certs.env
```

Comment:
Do we still even _have_ a `certs.env` (given secrets generator)?

Reply:
Huh. I do... but is that leftover from an old build? I guess it must be, and I never noticed because of the `.gitignore` :)

Fixed in b77b5eb.